src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -186,6 +189,15 @@ public int getChangedFiles() throws IOException {
     }
 
     /**
+     * FYI: https://developer.github.com/changes/2013-04-25-deprecating-merge-commit-sha
+     */
+    @Deprecated
+    public String getMergeCommitSha() throws IOException {
```

Comment:
Right. [This line](https://github.com/recena/github-api/blob/b0789a7ce72b6e1773b47d4e778535f839863480/src/main/java/org/kohsuke/github/GitHub.java#L564) is preventing to fail on missing properties.